### PR TITLE
added bugfix for showing images in card list for image right selection

### PR DESCRIPTION
**Pull request by astha94**

### CODE REVIEW

**Comment by astha94** [comment]
@scott-massey I pushed to master earlier and in that I pushed the changes you are requesting. I then reverted my changes from card list and added them to this branch.

**Comment by scott-massey** [comment]
Oh, ok makes sense!

**Comment by nucks** [comment]
[Setting "Image Right" for Media position on the Card List component doesn't work](https://trello.com/c/pYmbCjLK/2051-setting-image-right-for-media-position-on-the-card-list-component-doesnt-work)

**Review by zachbharris — CHANGES_REQUESTED** [review]
It looks like the image doesn't center / cover in the parent wrapper correctly. A simple change could be just to revert the current changes and just swap the order of the style prop array in the `<Image />` component on line `135`.

### Current
```jsx
<Image
  resizeMode="cover"
  source={source}
  style={[imageStyles, styles.image]}
/>
```

### Proposed Change

```jsx
<Image
  resizeMode="cover"
  source={source}
  style={[styles.image, imageStyles]}
/>
```

**Comment by zachbharris** [comment]
@benhaefele do we have a mockup with what the card list should like with image right? Currently the image is pushed up against the right side of the container. Should it have the same padding on the right as the rest of the content?

Screen Shot 2020-09-16 at 12 50 03 PM

**Comment by benhaefele** [comment]
It should probably match the padding at the top of the image

File: src/CardList/index.js [hunk 1]
```diff
@@ -384,11 +383,11 @@ const styles = StyleSheet.create({
...
   rightMedia: {
-    height: 80,
-    width: 80,
-    marginTop: 4,
-    marginBottom: 8,
+    height: 100,
+    width: 100,
+    marginBottom: -25,
     marginLeft: 16,
+    marginRight: -16,
```
Comment (zachbharris) [inline src/CardList/index.js:390]: Per Ben's response, we don't need to set the margin right here.

```suggestion
```

or

```suggestion
    marginRight: 0,
```

File: src/CardList/index.js [hunk 2]
```diff
@@ -384,11 +383,11 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
   },
   rightMedia: {
-    height: 80,
-    width: 80,
-    marginTop: 4,
-    marginBottom: 8,
+    height: 100,
+    width: 100,
```
Comment (zachbharris) [inline src/CardList/index.js:387]: What was the motivation for this change?

File: src/CardList/index.js [hunk 3]
```diff
@@ -384,11 +383,11 @@ const styles = StyleSheet.create({
...
+    marginBottom: -25,
```
Comment (zachbharris) [inline src/CardList/index.js:388]: What was the motivation for this change?

**Review by zachbharris — CHANGES_REQUESTED** [review]

File: src/CardList/index.js (hunk 3 above)
Comment (astha94) [reply src/CardList/index.js:388]: Earlier when it was set to previous CSS values it was resulting in the image not showing on the right. So I added those CSS changes. This was before you pointed out the change:   style={[styles.image, imageStyles]}
I can revert back to the old CSS properties and check if just the change you proposed was enough to show the images on the right.

**Review by zachbharris — APPROVED** [review]

**Pull request merged**